lib/node_modules/@stdlib/blas/ext/base/gfind-index/test/test.ndarray.js

```diff
@@ -0,0 +1,145 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var toAccessorArray = require( '@stdlib/array/base/to-accessor-array' );
+var gfindIndex = require( './../lib/ndarray.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof gfindIndex, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function has an arity of 6', function test( t ) {
+	t.strictEqual( gfindIndex.length, 6, 'has expected arity' );
+	t.end();
+});
+
+tape( 'the function returns the index of the first element which passes a test implemented by a predicate function', function test( t ) {
+	var actual;
+	var x;
+
+	x = [ 1.0, 1.0, 2.0, 3.0, 2.0, 3.0 ];
+
+	// Nonnegative stride...
+	actual = gfindIndex( x.length, x, 1, 3, clbk );
```

Comment:
Your `N` parameters here and below are incorrect. It should not be `x.length` when the `offset` is non-zero.